openlibrary/plugins/upstream/mybooks.py

```diff
@@ -272,6 +294,64 @@ def render(
             elif self.key == 'list':
                 docs = list
 
+            elif is_logged_in_user and self.key == "loan_history":
+                account = OpenLibraryAccount.get(username=logged_in_username)
+                s3_keys = web.ctx.site.store.get(account._key).get('s3_keys')
+                limit = RESULTS_PER_PAGE
+                offset = page * limit - limit
+
+                loan_history = s3_loan_api(
+                    # s3_keys=s3_keys, action='user_borrow_history'
+                    s3_keys=s3_keys,
+                    action='user_borrow_history',
+                    limit=limit + 1,
+                    offset=offset,
+                    # newest=True,
+                    # sort=True,
```

Comment:
```suggestion
                    newest=True,
```
I'll change it so staging uses a test version of the updated loan API, so we can use `newest=True` here.